fieldsolver/derivatives.cpp

```diff
@@ -364,25 +364,29 @@ void calculateDerivativesSimple(
    
    phiprof::stop(timer);
 
-   timer=phiprof::initializeTimer("Compute cells");
-   phiprof::start(timer);
+   //timer=phiprof::initializeTimer("Compute cells");
+   //phiprof::start(timer);
 
    // Calculate derivatives
-   #pragma omp parallel for collapse(3)
-   for (int k=0; k<gridDims[2]; k++) {
-      for (int j=0; j<gridDims[1]; j++) {
-         for (int i=0; i<gridDims[0]; i++) {
-            if (technicalGrid.get(i,j,k)->sysBoundaryFlag == sysboundarytype::DO_NOT_COMPUTE) continue;
-            if (RKCase == RK_ORDER1 || RKCase == RK_ORDER2_STEP2) {
-               calculateDerivatives(i,j,k, perBGrid, momentsGrid, dPerBGrid, dMomentsGrid, technicalGrid, sysBoundaries, RKCase);
-            } else {
-               calculateDerivatives(i,j,k, perBDt2Grid, momentsDt2Grid, dPerBGrid, dMomentsGrid, technicalGrid, sysBoundaries, RKCase);
+   #pragma omp parallel
+   {
+      phiprof::start("FS derivatives compute cells");
```

Comment:
Why not call this guy just outside of the threaded region?

Reply:
Because if the phiprof call is outside the threaded region, we don't get any threading balance information in phiprof. At least that's how I understand it.

Reply:
also, only if we open them like this do we see the thread imbalances properly in tau traces